build.gradle

```
@@ -165,6 +165,7 @@ release {
 
 dependencies {
     // Generates configuration metadata that IntelliJ can use
+    annotationProcessor('org.hibernate:hibernate-jpamodelgen:5.4.18.Final')
```

Version unneeded, it's already handled by the `org.springframework.boot:spring-boot-dependencies`
```suggestion
    annotationProcessor('org.hibernate:hibernate-jpamodelgen')
```